client/my-sites/checkout/composite-checkout/use-get-thank-you-url.js

```diff
@@ -362,6 +362,10 @@ export function useGetThankYouUrl( {
 		const orderId = transactionResult.order_id;
 		const isTransactionResultEmpty = isEmpty( transactionResult );
 
+		if ( isEmpty( siteSlug ) || 'no-user' === siteSlug ) {
```

Comment:
I think we can be pretty confident here that `siteSlug` is either a string or a falsy value, so rather than the rather powerful and complex `isEmpty`, we could change this to something like:

```js
if ( siteSlug === 'no-user' || ! siteSlug )
```

`isEmpty` would be useful if we expected an empty array or empty object.

Reply:
Done